ui/src/app/cluster-workflow-templates/components/cluster-workflow-template-link.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+import {uiUrl} from '../../shared/base';
+import {LinkButton} from '../../shared/components/link-button';
+
+export const ClusterWorkflowTemplateLink = (props: {name: string}) => (
```

Comment:
Why not generalize this component?

Reply:
I can't do this for cluster workflows - I did expect to re-use this, but I've not done so could be inlined?